### PR TITLE
Integrate department service with api

### LINKED ISSUE #50: Integrate department service with API

### PR DESCRIPTION
**Pull request by laska-sl**

### CODE REVIEW

File: ArcadiaParties-SPA/src/app/department/effects/effect.ts [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-            map(() => loadDepartmentsSuccessAction({ departments: this.departmentService.getDepartments() }))
-        )
-    );
+  loadDepartments = createEffect(() =>
+    this.actions.pipe(
+      ofType(loadDepartmentsAction),
+      mergeMap(() =>
+        this.departmentService
+          .getDepartments()
+          .pipe(map(departmentsFromService => loadDepartmentsSuccessAction({ departments: departmentsFromService })))
```
Comment (sponkrashin) [inline ArcadiaParties-SPA/src/app/department/effects/effect.ts]: You don't need to use another pipe, you can do all of this in top-level pipe to reduce nesting.

**Review by sponkrashin — CHANGES_REQUESTED** [review]

File: ArcadiaParties-SPA/src/app/department/effects/effect.ts (hunk 1 above)
Comment (laska-sl) [reply ArcadiaParties-SPA/src/app/department/effects/effect.ts]: Done according to your advice

**Review by sponkrashin — DISMISSED** [review]
Wait until base PR is merged. Then merge this PR to develop.

File: ArcadiaParties-SPA/src/environments/environment.ts [hunk 2]
```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  apiUrl: 'http://localhost:5000/api/'
```
Comment (sponkrashin) [inline ArcadiaParties-SPA/src/environments/environment.ts]: Not needed.

**Review by sponkrashin — CHANGES_REQUESTED** [review]

File: ArcadiaParties-SPA/src/environments/environment.ts (hunk 2 above)
Comment (laska-sl) [reply ArcadiaParties-SPA/src/environments/environment.ts]: Removed.

**Review by sponkrashin — APPROVED** [review]

**Pull request merged**